packages/cashier/src/Components/Form/payment-agent-transfer-form.jsx

```diff
@@ -151,7 +151,7 @@ const PaymentAgentTransferForm = ({
                                     className='payment-agent-transfer__input-area'
                                     type='textarea'
                                     label={localize('Description')}
-                                    error={touched.description && errors.description}
+                                    error={errors.description}
```

Comment:
Do we also need to make this change on `Amount` field to make it consistent or amount field validation should not be triggered before touch?

Reply:
in reality I tought the same like you but the reporter didnt report that. Should I just do it myself? Thanks as always akmal

Reply:
I think it's better to do it for another field as well to keep it consistent, otherwise, sooner or later another QA will report that again